File: CCCanvas.Plane.OC/CCCanvas.Plane.OC/ViewController.m
//
//  ViewController.m
//  CCCanvas.Plane.OC
//
//  Created by CC (deng you hua | cworld1000@gmail.com) on 2021/09/12.
//  https://github.com/ccworld1000/CCCanvas.Plane

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (CCCanvasPoint *)getCanvasPoint {
	 return [[CCCanvasPoint alloc] initPlaneWith4Point:CGPointMake(-0.5, 0.5) pointB:CGPointMake(-0.5, -0.5) pointC:CGPointMake(0.5, -0.5) pointD:CGPointMake(0.5, 0.5)];
}

- (void)viewDidLoad {
  [super viewDidLoad];
  // Do any additional setup after loading the view.
}


@end

File: CCCanvas.Plane.Swift/CCCanvas.Plane.Swift/ViewController.swift
//
//  ViewController.swift
//  CCCanvas.Plane.Swift
//
//  Created by CC (deng you hua | cworld1000@gmail.com) on 2021/8/15.
//  https://github.com/ccworld1000/CCCanvas.Plane

import UIKit
import CCCanvas

class ViewController: CCCanvasVC {
    override func getCanvasPoint() -> CCCanvasPoint {
        return CCCanvasPoint(planeWith4Point: CGPoint(x: -0.5, y: 0.5),
                             pointB: CGPoint(x: -0.5, y: -0.5),
                             pointC: CGPoint(x: 0.5, y: -0.5),
                             pointD: CGPoint(x: 0.5, y: 0.5))
    }

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }


}

